spec/controllers/admin/booths_controller_spec.rb

```diff
@@ -0,0 +1,149 @@
+require 'spec_helper'
+
+describe Admin::BoothsController do
+
+  let(:admin) { create(:admin) }
+  let(:conference) { create(:conference) }
+  let(:booth) { create(:booth, conference: conference) }
+  let(:admin) { create(:admin) }
+
+  context 'not logged in user' do
+
+    describe 'GET index' do
+      it 'does not render admin/booths#index' do
+        get :index, conference_id: conference.short_title
+        expect(response).to redirect_to(user_session_path)
+      end
+    end
+
+    describe 'GET show' do
+      it 'does not render admin/booths#show' do
```

Comment:
Later on, we probably want to change this, so that a not signed in user, can access show action for confirmed booths.